src/CloseButton.tsx

```diff
@@ -2,7 +2,7 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import classNames from 'classnames';
 
-export type CloseButtonVariant = 'white';
+export type CloseButtonVariant = string;
```

Comment:
I checked over the bootstrap sass files and it looks like they don't offer any additional customizing here like they did with button colors.  We can leave this as `white`.

Reply:
Oops, I didn't meant to leave this change in the code; I was just testing how the rendered docs would look and this must have been swept up in my change.  Will clear it out.